src/types/index.ts

```diff
@@ -263,6 +264,8 @@ export interface IAccountRaw {
   type: AccountType;
   isRestored: boolean;
   protocol: Protocol;
+  address?: AccountAddress;
+  airGapPublicKey?: string;
```

Comment:
Maybe this could be renamed to `publicKey` so we'll use this property in the future for accounts imported directly by entering the public key? There was such request from people.